dali/util/ocv.cc

```diff
@@ -13,58 +13,38 @@
 // limitations under the License.
 
 #include "dali/util/ocv.h"
+#include <algorithm>
 #include <cmath>
-#include <utility>
 #include <map>
-#include <algorithm>
 #include <tuple>
+#include <utility>
 #include "dali/core/error_handling.h"
 #include "dali/kernels/imgproc/color_manipulation/color_space_conversion_impl.h"
 
 namespace dali {
 
-int OCVInterpForDALIInterp(DALIInterpType type, int *ocv_type) {
-  switch (type) {
-  case DALI_INTERP_NN:
-    *ocv_type =  cv::INTER_NEAREST;
-    break;
-  case DALI_INTERP_LINEAR:
-    *ocv_type =  cv::INTER_LINEAR;
-    break;
-  case DALI_INTERP_CUBIC:
-    *ocv_type =  cv::INTER_CUBIC;
-    break;
-  default:
-    return DALIError;
-  }
-  return DALISuccess;
-}
-
 int GetOpenCvChannelType(size_t c) {
   return type2ocv<uint8_t>::value(c);
 }
 
-static cv::ColorConversionCodes GetOpenCvColorConversionCode(
-  DALIImageType input_type, DALIImageType output_type) {
-    using ColorConversionPair = std::pair<DALIImageType, DALIImageType>;
-    using ColorConversionMap = std::map< ColorConversionPair, cv::ColorConversionCodes >;
-    static const ColorConversionMap color_conversion_map = {
-        { {DALI_RGB, DALI_BGR},  cv::COLOR_RGB2BGR },
-        { {DALI_RGB, DALI_GRAY}, cv::COLOR_RGB2GRAY },
-
-        { {DALI_BGR, DALI_RGB},  cv::COLOR_BGR2RGB },
-        { {DALI_BGR, DALI_GRAY}, cv::COLOR_BGR2GRAY },
-
-        { {DALI_GRAY, DALI_RGB}, cv::COLOR_GRAY2RGB },
-        { {DALI_GRAY, DALI_BGR}, cv::COLOR_GRAY2BGR },
-    };
-
-    const ColorConversionPair color_conversion_pair{ input_type, output_type };
-    const auto it = color_conversion_map.find(color_conversion_pair);
-    if (it == color_conversion_map.end()) {
-      return cv::COLOR_COLORCVT_MAX;
-    }
-    return it->second;
+static cv::ColorConversionCodes GetOpenCvColorConversionCode(DALIImageType input_type,
+                                                             DALIImageType output_type) {
+  using ColorConversionPair = std::pair<DALIImageType, DALIImageType>;
+  using ColorConversionMap = std::map<ColorConversionPair, cv::ColorConversionCodes>;
+  static const ColorConversionMap color_conversion_map = {
+      {{DALI_RGB, DALI_BGR}, cv::COLOR_RGB2BGR},   {{DALI_RGB, DALI_GRAY}, cv::COLOR_RGB2GRAY},
+
+      {{DALI_BGR, DALI_RGB}, cv::COLOR_BGR2RGB},   {{DALI_BGR, DALI_GRAY}, cv::COLOR_BGR2GRAY},
+
+      {{DALI_GRAY, DALI_RGB}, cv::COLOR_GRAY2RGB}, {{DALI_GRAY, DALI_BGR}, cv::COLOR_GRAY2BGR},
+  };
+
+  const ColorConversionPair color_conversion_pair{input_type, output_type};
+  const auto it = color_conversion_map.find(color_conversion_pair);
+  if (it == color_conversion_map.end()) {
+    return cv::COLOR_COLORCVT_MAX;
+  }
+  return it->second;
```

Comment:
Please refrain from reformatting otherwise unchanged code.

Reply:
For structures like this, I can wrap in disable clang-fromat context (like I just did). But for the rest of the code, I think it's good to continue incrementally rolling-out consistent formatting.

Lmk my clang-format is doing something wrong, if its not using the root .clang-format style correctly, I'll look into it.

For example, the above code previously had random whitespace in the template args 😆.
```
using ColorConversionPair = std::pair<DALIImageType, DALIImageType>;
using ColorConversionMap = std::map< ColorConversionPair, cv::ColorConversionCodes >;
```